components/event-publisher-proxy/pkg/sender/jetstream/jetstream.go

```diff
@@ -26,6 +26,7 @@ const (
 	natsBackend           = "nats"
 	jestreamHandlerName   = "jetstream-handler"
 	noSpaceLeftErrMessage = "no space left on device"
+	natsMaxBytesExceeded  = "nats: maximum bytes exceeded"
```

Comment:
can we use the `ErrMaxBytes` from nats-server library and use the n`errors.Is` instead of checking for error text ?

Reply:
No we cannot, but i improved the code to evaluate the errorcode returned by the api